code/modules/research/techweb/all_nodes.dm

```diff
@@ -222,7 +222,7 @@
 	id = "posibrain"
 	display_name = "Positronic Brain"
 	description = "Applied usage of neural technology allowing for autonomous AI units based on special metallic cubes with conductive and processing circuits."
-	prereq_ids = list("mmi", "neural_programming")
+	prereq_ids = list("neural_programming", "mmi")

```

Comment:
why did you reverse the order here?

Reply:
doesnt change anything

Reply:
I know, just curious

Reply:
He had previously removed the prerequisites, but readded them in this commit by suggestion, that's why.

To be honest, the order kinda makes a little more sense this way, anyway, IMO.